### PR TITLE
Added configuration and getting started information for the sdk.

### PR DESCRIPTION
**Pull request by rakker91**
Renamed the proxy configuration variables.

Issue Number: 228

## Summary
Adds configuration documentation to the repo.

## Changes

- Added a GettingStarted.md file and pointed the current ReadMe.md file to it
- Renamed the web proxy configuration variables to make them less verbose.

## Checklist

- [x] Local Tests Passing?
- [x] CICD and Pipeline Tests Passing?
- [x] Added any new Tests?
- [x] Documentation Updates Made?
- [x] This is not a breaking change. If it is, please describe it below.

## Is this a breaking change?
Yes.  Configuration variables have been renamed.

## Anything else?
This PR Closes Issue #298

### CODE REVIEW

**Review by juzuluag — APPROVED** [review]
👍

File: GettingStarted.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+```bash
+  CarbonAwareVars__CarbonIntensityDataSource="WattTime"
+```
+
+Note that double underscores are used to represent dotted notation or child elements that you see in the JSON below.  For example, to set proxy information using environment variables, you'd do this:
+
+```bash
+  CarbonAwareVars__Proxy__UseProxy
+```
+
```
Comment (juzuluag) [inline GettingStarted.md:36]: Suggestion: Add a complete set of env vars (aiming for a AKS env or run it on vscode) needed in order to go through the proxy and hit watttime data src.

**Review by JenMadiedo — APPROVED** [review]
🎉 this is great!

File: GettingStarted.md (hunk 1 above)
Comment (rakker91) [reply GettingStarted.md:36]: Good idea.  I'll add it.

**Pull request merged**